content/pt/docs/concepts/storage/persistent-volumes.md

```diff
@@ -0,0 +1,751 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- thockin
+- msau42
+- xing-yang
+título: Persistentes Volumes
+funcionalidade:
+  title: Orquestração de Storage
+  descrição: >
+    Mountar automaticamente o storage de sua escolha, seja de um storage local, de um provedor de cloud pública, como <a href="https://cloud.google.com/storage/">GCP</a> ou <a href="https://aws.amazon.com/products/storage/">AWS</a>, ou um storage de rede, como NFS, iSCSI, Gluster, Ceph, Cinder ou Flocker.
```

Comment:
Some notes:

- "Mountar" can be "montar"
- We can translate "storage" to "armazenamento"?

Reply:
About "mountar" it was a lack of attention. I'll correct it. I don't see any problem translating storage to armazenamento.

Reply:
Hi Emanuel, it's not a problem change, it's because in Brazil we have a lot of peoples that do not know English, we need to think everybody, and need to minimize to use of English Terms. Please, change the word "storage" and to your practices in translation, use the minimum as possible English terms.
Thanks

Reply:
Emanuel, it's not a person, but we need to include all the peoples, beginners and not English speakers.